e2e/test/scenarios/actions/model-actions.cy.spec.js

```diff
@@ -328,6 +333,51 @@ describe(
       cy.findByLabelText("101").should("not.exist");
       cy.findByLabelText("ID").should("be.visible");
     });
+
+    it("should show detailed form errors for constraint violations when executing model actions", () => {
+      const actionName = "Update";
+
+      cy.get("@modelId").then(modelId => {
+        cy.visit(`/model/${modelId}/detail`);
+        cy.wait("@getModel");
+      });
+
+      cy.findByRole("tablist").within(() => {
+        cy.findByText("Actions").click();
+      });
```

Comment:
nit: makes this more concise and easier to read by chaining when there's only 1 action under `within`

```suggestion
      cy.findByRole("tablist").findByText("Actions").click();
```

Reply:
2b15d53b746fe9098a13f4e2694dd51a719a1ef2